2024-03-13 20:37:53,607 INFO    StreamThr :28048 [internal.py:wandb_internal():86] W&B internal server running at pid: 28048, started at: 2024-03-13 20:37:53.606788
2024-03-13 20:37:53,608 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status
2024-03-13 20:37:53,614 INFO    WriterThread:28048 [datastore.py:open_for_write():85] open: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\run-wwg0j0vs.wandb
2024-03-13 20:37:53,615 DEBUG   SenderThread:28048 [sender.py:send():380] send: header
2024-03-13 20:37:53,615 DEBUG   SenderThread:28048 [sender.py:send():380] send: run
2024-03-13 20:37:54,609 INFO    SenderThread:28048 [dir_watcher.py:__init__():211] watching files in: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files
2024-03-13 20:37:54,609 INFO    SenderThread:28048 [sender.py:_start_run_threads():1122] run started: wwg0j0vs with start time 1710333473.607787
2024-03-13 20:37:54,627 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: check_version
2024-03-13 20:37:54,628 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: check_version
2024-03-13 20:37:56,751 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: run_start
2024-03-13 20:37:56,792 DEBUG   HandlerThread:28048 [system_info.py:__init__():32] System info init
2024-03-13 20:37:56,793 DEBUG   HandlerThread:28048 [system_info.py:__init__():47] System info init done
2024-03-13 20:37:56,793 INFO    HandlerThread:28048 [system_monitor.py:start():181] Starting system monitor
2024-03-13 20:37:56,793 INFO    SystemMonitor:28048 [system_monitor.py:_start():145] Starting system asset monitoring threads
2024-03-13 20:37:56,793 INFO    HandlerThread:28048 [system_monitor.py:probe():201] Collecting system info
2024-03-13 20:37:56,798 INFO    SystemMonitor:28048 [interfaces.py:start():190] Started cpu monitoring
2024-03-13 20:37:56,799 INFO    SystemMonitor:28048 [interfaces.py:start():190] Started disk monitoring
2024-03-13 20:37:56,800 INFO    SystemMonitor:28048 [interfaces.py:start():190] Started gpu monitoring
2024-03-13 20:37:56,800 INFO    SystemMonitor:28048 [interfaces.py:start():190] Started memory monitoring
2024-03-13 20:37:56,814 INFO    SystemMonitor:28048 [interfaces.py:start():190] Started network monitoring
2024-03-13 20:37:56,836 DEBUG   HandlerThread:28048 [system_info.py:probe():196] Probing system
2024-03-13 20:37:56,837 DEBUG   HandlerThread:28048 [system_info.py:probe():244] Probing system done
2024-03-13 20:37:56,837 DEBUG   HandlerThread:28048 [system_monitor.py:probe():210] {'os': 'Windows-10-10.0.22000-SP0', 'python': '3.10.9', 'heartbeatAt': '2024-03-13T12:37:56.836770', 'startedAt': '2024-03-13T12:37:53.592784', 'docker': None, 'cuda': None, 'args': ('--task=P', '--weather=snowy'), 'state': 'running', 'program': 'E:\\自动驾驶\\车位检测\\parking-space-Identification-in-different-weather-main\\train.py', 'codePathLocal': 'train.py', 'codePath': 'train.py', 'host': 'LAPTOP-QIVT4NMT', 'username': 'lenovo', 'executable': 'E:\\anaconda\\python.exe', 'cpu_count': 8, 'cpu_count_logical': 16, 'cpu_freq': {'current': 3201.0, 'min': 0.0, 'max': 3201.0}, 'cpu_freq_per_core': [{'current': 3201.0, 'min': 0.0, 'max': 3201.0}], 'disk': {'/': {'total': 465.74511337280273, 'used': 178.51596069335938}}, 'gpu': 'NVIDIA GeForce RTX 3060 Laptop GPU', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 3060 Laptop GPU', 'memory_total': 6442450944}], 'memory': {'total': 15.855525970458984}}
2024-03-13 20:37:56,837 INFO    HandlerThread:28048 [system_monitor.py:probe():211] Finished collecting system info
2024-03-13 20:37:56,837 INFO    HandlerThread:28048 [system_monitor.py:probe():214] Publishing system info
2024-03-13 20:37:56,837 DEBUG   HandlerThread:28048 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-03-13 20:37:56,838 DEBUG   HandlerThread:28048 [system_info.py:_save_pip():68] Saving pip packages done
2024-03-13 20:37:56,838 DEBUG   HandlerThread:28048 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2024-03-13 20:37:56,850 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:37:57,643 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_created():271] file/dir created: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\requirements.txt
2024-03-13 20:37:57,643 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_created():271] file/dir created: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\conda-environment.yaml
2024-03-13 20:37:58,880 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:00,915 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:02,951 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:03,680 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\conda-environment.yaml
2024-03-13 20:38:03,770 DEBUG   HandlerThread:28048 [system_info.py:_save_conda():87] Saving conda packages done
2024-03-13 20:38:03,771 INFO    HandlerThread:28048 [system_monitor.py:probe():216] Finished publishing system info
2024-03-13 20:38:03,792 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:38:03,792 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: keepalive
2024-03-13 20:38:03,792 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:38:03,792 DEBUG   SenderThread:28048 [sender.py:send():380] send: files
2024-03-13 20:38:03,792 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-metadata.json with policy now
2024-03-13 20:38:03,805 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:38:03,806 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:38:03,807 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:38:04,687 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\conda-environment.yaml
2024-03-13 20:38:04,688 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_created():271] file/dir created: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-metadata.json
2024-03-13 20:38:04,989 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:06,125 DEBUG   SenderThread:28048 [sender.py:send():380] send: telemetry
2024-03-13 20:38:06,693 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_created():271] file/dir created: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:38:07,028 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:08,706 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:38:09,073 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:09,148 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:38:10,997 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:10,999 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:10,999 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:11,000 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:38:11,109 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:11,720 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_created():271] file/dir created: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:38:12,724 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:38:12,904 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:12,905 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:12,906 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:12,906 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:12,907 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:38:12,907 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:12,907 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:12,908 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:38:13,150 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:13,727 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:38:14,730 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:38:14,921 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:38:15,190 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:17,102 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:17,103 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:17,103 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:17,104 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:38:17,226 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:17,764 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:38:18,176 ERROR   wandb-upload_0:28048 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/wwg0j0vs/wandb-metadata.json?Expires=1710419888&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=vb4U4JQNqQUXpObZoiI423R5eeLXqNsz0bWPMXMh%2BS9kirBCIP%2FCiNP5SzJj9WHE1kMIpejLZu%2BD5bVQz%2BWkOQW6ReqVJ6dkelH5TDDN%2Fe7z%2F8jQkykphj4bxOHfPCr3xNQdY8oJDo4aBVTnAfVqFFYzQIo1QryvrATPQ6UUU%2FJBIuaZXwDUgPT%2BvcPlzP2giwtujrJTfs1RJgwC%2BUNlW0mxBA5ZomckKG%2BkkJ88KsT5O21ZLpzcFlJGqVURC1IzJAKbz3hbKXChORj7hgLV9S2r616JjFug7YcpX12GUJrrfDlLx%2FBmrcGgA94DFso0zW%2BfA8%2BPBg4oEF0LGfO31A%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/wwg0j0vs/wandb-metadata.json?Expires=1710419888&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=vb4U4JQNqQUXpObZoiI423R5eeLXqNsz0bWPMXMh%2BS9kirBCIP%2FCiNP5SzJj9WHE1kMIpejLZu%2BD5bVQz%2BWkOQW6ReqVJ6dkelH5TDDN%2Fe7z%2F8jQkykphj4bxOHfPCr3xNQdY8oJDo4aBVTnAfVqFFYzQIo1QryvrATPQ6UUU%2FJBIuaZXwDUgPT%2BvcPlzP2giwtujrJTfs1RJgwC%2BUNlW0mxBA5ZomckKG%2BkkJ88KsT5O21ZLpzcFlJGqVURC1IzJAKbz3hbKXChORj7hgLV9S2r616JjFug7YcpX12GUJrrfDlLx%2FBmrcGgA94DFso0zW%2BfA8%2BPBg4oEF0LGfO31A%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001F7BEA3C400>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:38:18,176 ERROR   wandb-upload_0:28048 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1321'}
2024-03-13 20:38:18,176 ERROR   wandb-upload_0:28048 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:38:18,779 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:38:18,810 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:38:18,810 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:38:18,810 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:38:18,958 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:18,959 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:19,205 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:19,205 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:19,206 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:38:19,207 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:19,207 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:19,208 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:38:19,259 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:19,781 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:38:20,210 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:38:20,797 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:38:21,289 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:23,169 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:23,170 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:23,170 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:23,171 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:38:23,326 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:23,804 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:38:24,805 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:38:25,110 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:25,111 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:25,111 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:25,112 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:25,113 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:38:25,113 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:25,113 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:25,114 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:38:25,358 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:25,808 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:38:26,129 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:38:26,810 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:38:26,811 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\config.yaml
2024-03-13 20:38:27,452 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:29,285 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:29,285 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:29,286 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:29,287 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:38:29,475 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:29,826 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:38:30,831 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:38:31,177 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:31,178 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:31,178 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:31,179 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:31,180 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:38:31,180 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:31,180 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:31,181 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:38:31,505 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:31,845 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:38:32,065 ERROR   wandb-upload_0:28048 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/wwg0j0vs/wandb-metadata.json?Expires=1710419888&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=vb4U4JQNqQUXpObZoiI423R5eeLXqNsz0bWPMXMh%2BS9kirBCIP%2FCiNP5SzJj9WHE1kMIpejLZu%2BD5bVQz%2BWkOQW6ReqVJ6dkelH5TDDN%2Fe7z%2F8jQkykphj4bxOHfPCr3xNQdY8oJDo4aBVTnAfVqFFYzQIo1QryvrATPQ6UUU%2FJBIuaZXwDUgPT%2BvcPlzP2giwtujrJTfs1RJgwC%2BUNlW0mxBA5ZomckKG%2BkkJ88KsT5O21ZLpzcFlJGqVURC1IzJAKbz3hbKXChORj7hgLV9S2r616JjFug7YcpX12GUJrrfDlLx%2FBmrcGgA94DFso0zW%2BfA8%2BPBg4oEF0LGfO31A%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/wwg0j0vs/wandb-metadata.json?Expires=1710419888&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=vb4U4JQNqQUXpObZoiI423R5eeLXqNsz0bWPMXMh%2BS9kirBCIP%2FCiNP5SzJj9WHE1kMIpejLZu%2BD5bVQz%2BWkOQW6ReqVJ6dkelH5TDDN%2Fe7z%2F8jQkykphj4bxOHfPCr3xNQdY8oJDo4aBVTnAfVqFFYzQIo1QryvrATPQ6UUU%2FJBIuaZXwDUgPT%2BvcPlzP2giwtujrJTfs1RJgwC%2BUNlW0mxBA5ZomckKG%2BkkJ88KsT5O21ZLpzcFlJGqVURC1IzJAKbz3hbKXChORj7hgLV9S2r616JjFug7YcpX12GUJrrfDlLx%2FBmrcGgA94DFso0zW%2BfA8%2BPBg4oEF0LGfO31A%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001F7BEAEB280>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:38:32,065 ERROR   wandb-upload_0:28048 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1321'}
2024-03-13 20:38:32,065 ERROR   wandb-upload_0:28048 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:38:32,195 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:38:32,852 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:38:33,542 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:33,805 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:38:33,806 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:38:33,821 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:38:35,388 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:35,564 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:35,607 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:35,607 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:35,608 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:38:35,863 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:38:36,870 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:38:37,322 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:38:37,322 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:37,323 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:37,324 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:37,324 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:37,325 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:38:37,325 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:37,325 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:37,326 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:38:37,601 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:37,885 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:38:38,887 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:38:39,640 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:41,546 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:41,547 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:41,547 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:41,548 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:38:41,680 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:41,919 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:38:42,552 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:38:42,935 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:38:43,425 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:43,426 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:43,427 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:43,427 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:43,428 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:38:43,428 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:43,428 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:43,429 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:38:43,716 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:43,937 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:38:44,950 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:38:45,743 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:46,481 ERROR   wandb-upload_0:28048 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/wwg0j0vs/wandb-metadata.json?Expires=1710419888&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=vb4U4JQNqQUXpObZoiI423R5eeLXqNsz0bWPMXMh%2BS9kirBCIP%2FCiNP5SzJj9WHE1kMIpejLZu%2BD5bVQz%2BWkOQW6ReqVJ6dkelH5TDDN%2Fe7z%2F8jQkykphj4bxOHfPCr3xNQdY8oJDo4aBVTnAfVqFFYzQIo1QryvrATPQ6UUU%2FJBIuaZXwDUgPT%2BvcPlzP2giwtujrJTfs1RJgwC%2BUNlW0mxBA5ZomckKG%2BkkJ88KsT5O21ZLpzcFlJGqVURC1IzJAKbz3hbKXChORj7hgLV9S2r616JjFug7YcpX12GUJrrfDlLx%2FBmrcGgA94DFso0zW%2BfA8%2BPBg4oEF0LGfO31A%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/wwg0j0vs/wandb-metadata.json?Expires=1710419888&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=vb4U4JQNqQUXpObZoiI423R5eeLXqNsz0bWPMXMh%2BS9kirBCIP%2FCiNP5SzJj9WHE1kMIpejLZu%2BD5bVQz%2BWkOQW6ReqVJ6dkelH5TDDN%2Fe7z%2F8jQkykphj4bxOHfPCr3xNQdY8oJDo4aBVTnAfVqFFYzQIo1QryvrATPQ6UUU%2FJBIuaZXwDUgPT%2BvcPlzP2giwtujrJTfs1RJgwC%2BUNlW0mxBA5ZomckKG%2BkkJ88KsT5O21ZLpzcFlJGqVURC1IzJAKbz3hbKXChORj7hgLV9S2r616JjFug7YcpX12GUJrrfDlLx%2FBmrcGgA94DFso0zW%2BfA8%2BPBg4oEF0LGfO31A%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001F7BEB93820>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:38:46,481 ERROR   wandb-upload_0:28048 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1321'}
2024-03-13 20:38:46,481 ERROR   wandb-upload_0:28048 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:38:46,481 INFO    wandb-upload_0:28048 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "E:\anaconda\lib\site-packages\urllib3\connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "E:\anaconda\lib\site-packages\urllib3\util\connection.py", line 95, in create_connection
    raise err
  File "E:\anaconda\lib\site-packages\urllib3\util\connection.py", line 85, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [WinError 10061] 由于目标计算机积极拒绝，无法连接。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\anaconda\lib\site-packages\urllib3\connectionpool.py", line 714, in urlopen
    httplib_response = self._make_request(
  File "E:\anaconda\lib\site-packages\urllib3\connectionpool.py", line 403, in _make_request
    self._validate_conn(conn)
  File "E:\anaconda\lib\site-packages\urllib3\connectionpool.py", line 1053, in _validate_conn
    conn.connect()
  File "E:\anaconda\lib\site-packages\urllib3\connection.py", line 363, in connect
    self.sock = conn = self._new_conn()
  File "E:\anaconda\lib\site-packages\urllib3\connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x000001F7BEB93820>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\anaconda\lib\site-packages\requests\adapters.py", line 487, in send
    resp = conn.urlopen(
  File "E:\anaconda\lib\site-packages\urllib3\connectionpool.py", line 798, in urlopen
    retries = retries.increment(
  File "E:\anaconda\lib\site-packages\urllib3\util\retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/wwg0j0vs/wandb-metadata.json?Expires=1710419888&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=vb4U4JQNqQUXpObZoiI423R5eeLXqNsz0bWPMXMh%2BS9kirBCIP%2FCiNP5SzJj9WHE1kMIpejLZu%2BD5bVQz%2BWkOQW6ReqVJ6dkelH5TDDN%2Fe7z%2F8jQkykphj4bxOHfPCr3xNQdY8oJDo4aBVTnAfVqFFYzQIo1QryvrATPQ6UUU%2FJBIuaZXwDUgPT%2BvcPlzP2giwtujrJTfs1RJgwC%2BUNlW0mxBA5ZomckKG%2BkkJ88KsT5O21ZLpzcFlJGqVURC1IzJAKbz3hbKXChORj7hgLV9S2r616JjFug7YcpX12GUJrrfDlLx%2FBmrcGgA94DFso0zW%2BfA8%2BPBg4oEF0LGfO31A%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001F7BEB93820>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\anaconda\lib\site-packages\wandb\sdk\internal\internal_api.py", line 2549, in upload_file
    response = self._upload_file_session.put(
  File "E:\anaconda\lib\site-packages\requests\sessions.py", line 647, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "E:\anaconda\lib\site-packages\requests\sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "E:\anaconda\lib\site-packages\requests\sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "E:\anaconda\lib\site-packages\requests\adapters.py", line 520, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/wwg0j0vs/wandb-metadata.json?Expires=1710419888&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=vb4U4JQNqQUXpObZoiI423R5eeLXqNsz0bWPMXMh%2BS9kirBCIP%2FCiNP5SzJj9WHE1kMIpejLZu%2BD5bVQz%2BWkOQW6ReqVJ6dkelH5TDDN%2Fe7z%2F8jQkykphj4bxOHfPCr3xNQdY8oJDo4aBVTnAfVqFFYzQIo1QryvrATPQ6UUU%2FJBIuaZXwDUgPT%2BvcPlzP2giwtujrJTfs1RJgwC%2BUNlW0mxBA5ZomckKG%2BkkJ88KsT5O21ZLpzcFlJGqVURC1IzJAKbz3hbKXChORj7hgLV9S2r616JjFug7YcpX12GUJrrfDlLx%2FBmrcGgA94DFso0zW%2BfA8%2BPBg4oEF0LGfO31A%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001F7BEB93820>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\anaconda\lib\site-packages\wandb\sdk\lib\retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "E:\anaconda\lib\site-packages\wandb\sdk\internal\internal_api.py", line 2580, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "E:\anaconda\lib\site-packages\wandb\sdk\internal\internal_api.py", line 2549, in upload_file
    response = self._upload_file_session.put(
  File "E:\anaconda\lib\site-packages\requests\sessions.py", line 647, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "E:\anaconda\lib\site-packages\requests\sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "E:\anaconda\lib\site-packages\requests\sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "E:\anaconda\lib\site-packages\requests\adapters.py", line 520, in send
    raise ConnectionError(e, request=request)
wandb.sdk.lib.retry.TransientError: None
2024-03-13 20:38:47,618 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:38:47,618 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:47,619 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:47,619 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:47,620 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:38:47,779 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:47,970 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:38:48,812 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:38:48,813 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:38:48,828 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:38:48,985 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:38:49,559 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:49,560 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:49,561 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:49,561 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:49,562 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:38:49,562 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:49,562 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:49,563 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:38:49,818 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:49,991 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:38:51,000 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:38:51,844 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:53,585 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:38:53,740 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:53,740 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:53,740 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:53,741 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:38:53,875 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:54,018 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:38:55,024 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:38:55,601 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:55,602 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:55,603 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:55,603 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:55,604 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:38:55,604 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:55,604 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:55,605 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:38:55,915 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:56,029 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:38:56,843 DEBUG   SystemMonitor:28048 [system_monitor.py:_start():159] Starting system metrics aggregation loop
2024-03-13 20:38:56,844 DEBUG   SenderThread:28048 [sender.py:send():380] send: stats
2024-03-13 20:38:57,036 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:38:57,995 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:38:58,854 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:38:59,808 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:38:59,809 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:38:59,809 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:38:59,810 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:00,020 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:00,051 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:39:01,054 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:39:01,712 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:01,713 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:01,713 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:01,714 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:01,715 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:01,715 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:01,715 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:01,716 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:02,052 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:02,060 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:39:03,064 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:39:03,139 ERROR   wandb-upload_0:28048 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/wwg0j0vs/wandb-metadata.json?Expires=1710419888&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=vb4U4JQNqQUXpObZoiI423R5eeLXqNsz0bWPMXMh%2BS9kirBCIP%2FCiNP5SzJj9WHE1kMIpejLZu%2BD5bVQz%2BWkOQW6ReqVJ6dkelH5TDDN%2Fe7z%2F8jQkykphj4bxOHfPCr3xNQdY8oJDo4aBVTnAfVqFFYzQIo1QryvrATPQ6UUU%2FJBIuaZXwDUgPT%2BvcPlzP2giwtujrJTfs1RJgwC%2BUNlW0mxBA5ZomckKG%2BkkJ88KsT5O21ZLpzcFlJGqVURC1IzJAKbz3hbKXChORj7hgLV9S2r616JjFug7YcpX12GUJrrfDlLx%2FBmrcGgA94DFso0zW%2BfA8%2BPBg4oEF0LGfO31A%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/wwg0j0vs/wandb-metadata.json?Expires=1710419888&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=vb4U4JQNqQUXpObZoiI423R5eeLXqNsz0bWPMXMh%2BS9kirBCIP%2FCiNP5SzJj9WHE1kMIpejLZu%2BD5bVQz%2BWkOQW6ReqVJ6dkelH5TDDN%2Fe7z%2F8jQkykphj4bxOHfPCr3xNQdY8oJDo4aBVTnAfVqFFYzQIo1QryvrATPQ6UUU%2FJBIuaZXwDUgPT%2BvcPlzP2giwtujrJTfs1RJgwC%2BUNlW0mxBA5ZomckKG%2BkkJ88KsT5O21ZLpzcFlJGqVURC1IzJAKbz3hbKXChORj7hgLV9S2r616JjFug7YcpX12GUJrrfDlLx%2FBmrcGgA94DFso0zW%2BfA8%2BPBg4oEF0LGfO31A%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001F7BEAEAAA0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:39:03,139 ERROR   wandb-upload_0:28048 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1321'}
2024-03-13 20:39:03,139 ERROR   wandb-upload_0:28048 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:39:03,827 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:39:03,827 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:39:03,828 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:39:04,090 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:05,902 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:05,909 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:05,909 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:39:05,909 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:05,910 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:06,072 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:39:06,122 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:07,075 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:39:07,776 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:07,776 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:07,777 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:07,777 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:07,778 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:07,779 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:07,779 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:07,780 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:08,107 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:39:08,154 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:09,112 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:39:10,183 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:11,829 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:39:12,211 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:12,319 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:12,320 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:12,320 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:12,321 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:13,197 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:39:13,198 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:39:14,246 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:14,270 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:14,271 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:14,271 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:14,272 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:14,272 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:14,273 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:14,273 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:14,274 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:15,228 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:39:15,229 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:39:16,281 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:17,284 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:39:18,315 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:18,517 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:18,518 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:18,518 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:18,519 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:18,826 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:39:18,827 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:39:18,843 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:39:19,315 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:39:20,357 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:20,456 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:20,457 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:20,458 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:20,458 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:20,459 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:20,459 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:20,460 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:20,460 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:21,345 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:39:21,346 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:39:22,391 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:22,487 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:39:23,377 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:39:24,332 ERROR   wandb-upload_0:28048 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/wwg0j0vs/wandb-metadata.json?Expires=1710419888&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=vb4U4JQNqQUXpObZoiI423R5eeLXqNsz0bWPMXMh%2BS9kirBCIP%2FCiNP5SzJj9WHE1kMIpejLZu%2BD5bVQz%2BWkOQW6ReqVJ6dkelH5TDDN%2Fe7z%2F8jQkykphj4bxOHfPCr3xNQdY8oJDo4aBVTnAfVqFFYzQIo1QryvrATPQ6UUU%2FJBIuaZXwDUgPT%2BvcPlzP2giwtujrJTfs1RJgwC%2BUNlW0mxBA5ZomckKG%2BkkJ88KsT5O21ZLpzcFlJGqVURC1IzJAKbz3hbKXChORj7hgLV9S2r616JjFug7YcpX12GUJrrfDlLx%2FBmrcGgA94DFso0zW%2BfA8%2BPBg4oEF0LGfO31A%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/wwg0j0vs/wandb-metadata.json?Expires=1710419888&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=vb4U4JQNqQUXpObZoiI423R5eeLXqNsz0bWPMXMh%2BS9kirBCIP%2FCiNP5SzJj9WHE1kMIpejLZu%2BD5bVQz%2BWkOQW6ReqVJ6dkelH5TDDN%2Fe7z%2F8jQkykphj4bxOHfPCr3xNQdY8oJDo4aBVTnAfVqFFYzQIo1QryvrATPQ6UUU%2FJBIuaZXwDUgPT%2BvcPlzP2giwtujrJTfs1RJgwC%2BUNlW0mxBA5ZomckKG%2BkkJ88KsT5O21ZLpzcFlJGqVURC1IzJAKbz3hbKXChORj7hgLV9S2r616JjFug7YcpX12GUJrrfDlLx%2FBmrcGgA94DFso0zW%2BfA8%2BPBg4oEF0LGfO31A%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001F7BEB90760>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:39:24,333 ERROR   wandb-upload_0:28048 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1321'}
2024-03-13 20:39:24,333 ERROR   wandb-upload_0:28048 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:39:24,430 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:24,682 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:24,683 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:24,683 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:24,684 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:25,419 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:39:26,451 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:39:26,468 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:26,626 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:26,626 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:26,627 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:26,628 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:26,629 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:26,629 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:26,629 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:26,630 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:26,851 DEBUG   SenderThread:28048 [sender.py:send():380] send: stats
2024-03-13 20:39:27,456 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:39:27,854 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:39:28,483 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:39:28,553 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:30,571 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:30,928 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:30,928 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:30,928 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:30,929 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:31,537 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:39:32,557 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:39:32,592 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:32,873 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:39:32,873 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:32,874 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:32,875 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:32,875 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:32,876 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:32,877 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:32,877 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:32,877 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:33,572 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:39:33,823 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:39:33,824 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:39:33,855 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:39:34,598 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:39:34,624 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:36,647 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:37,106 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:37,107 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:37,107 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:37,108 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:37,644 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:39:38,119 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:39:38,670 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:39:38,689 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:39,018 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:39,019 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:39,020 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:39,020 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:39,021 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:39,021 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:39,021 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:39,022 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:39,679 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:39:40,711 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:39:40,727 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:42,763 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:43,232 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:39:43,232 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:43,233 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:43,233 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:43,234 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:43,759 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:39:44,779 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:39:44,797 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:45,201 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:45,202 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:45,202 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:45,203 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:45,203 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:45,204 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:45,204 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:45,205 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:45,784 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:39:46,819 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:39:46,843 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:48,235 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:39:48,851 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:39:48,851 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:39:48,871 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:39:48,873 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:49,489 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:49,489 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:49,490 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:49,490 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:49,862 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:39:50,895 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:39:50,911 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:51,457 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:51,457 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:51,458 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:51,459 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:51,459 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:51,460 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:51,460 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:51,461 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:51,904 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:39:52,936 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:39:52,952 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:53,475 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:39:54,027 ERROR   wandb-upload_0:28048 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/wwg0j0vs/wandb-metadata.json?Expires=1710419888&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=vb4U4JQNqQUXpObZoiI423R5eeLXqNsz0bWPMXMh%2BS9kirBCIP%2FCiNP5SzJj9WHE1kMIpejLZu%2BD5bVQz%2BWkOQW6ReqVJ6dkelH5TDDN%2Fe7z%2F8jQkykphj4bxOHfPCr3xNQdY8oJDo4aBVTnAfVqFFYzQIo1QryvrATPQ6UUU%2FJBIuaZXwDUgPT%2BvcPlzP2giwtujrJTfs1RJgwC%2BUNlW0mxBA5ZomckKG%2BkkJ88KsT5O21ZLpzcFlJGqVURC1IzJAKbz3hbKXChORj7hgLV9S2r616JjFug7YcpX12GUJrrfDlLx%2FBmrcGgA94DFso0zW%2BfA8%2BPBg4oEF0LGfO31A%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/wwg0j0vs/wandb-metadata.json?Expires=1710419888&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=vb4U4JQNqQUXpObZoiI423R5eeLXqNsz0bWPMXMh%2BS9kirBCIP%2FCiNP5SzJj9WHE1kMIpejLZu%2BD5bVQz%2BWkOQW6ReqVJ6dkelH5TDDN%2Fe7z%2F8jQkykphj4bxOHfPCr3xNQdY8oJDo4aBVTnAfVqFFYzQIo1QryvrATPQ6UUU%2FJBIuaZXwDUgPT%2BvcPlzP2giwtujrJTfs1RJgwC%2BUNlW0mxBA5ZomckKG%2BkkJ88KsT5O21ZLpzcFlJGqVURC1IzJAKbz3hbKXChORj7hgLV9S2r616JjFug7YcpX12GUJrrfDlLx%2FBmrcGgA94DFso0zW%2BfA8%2BPBg4oEF0LGfO31A%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001F7BEB90880>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:39:54,027 ERROR   wandb-upload_0:28048 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1321'}
2024-03-13 20:39:54,027 ERROR   wandb-upload_0:28048 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:39:54,998 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:55,692 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:55,692 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:55,692 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:55,694 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:55,989 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:39:56,860 DEBUG   SenderThread:28048 [sender.py:send():380] send: stats
2024-03-13 20:39:57,016 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:39:57,038 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:39:57,606 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:57,607 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:39:57,608 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:57,608 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:57,609 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:57,609 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:39:57,610 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:39:57,610 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:39:58,018 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:39:58,618 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:39:59,068 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:39:59,139 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:40:01,164 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:40:01,892 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:40:01,893 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:40:01,893 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:40:01,894 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:40:02,116 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:40:03,148 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:40:03,183 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:40:03,810 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:40:03,810 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:40:03,811 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:40:03,812 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:40:03,813 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:40:03,814 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:40:03,814 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:40:03,814 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:40:03,815 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:40:03,837 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:40:03,837 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:40:03,885 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:40:04,155 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:40:05,190 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:40:05,207 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:40:07,253 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:40:08,108 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:40:08,109 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:40:08,109 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:40:08,110 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:40:08,239 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:40:09,116 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:40:09,273 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:40:09,293 ERROR   gpu       :28048 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:40:10,002 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:40:10,002 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:40:10,003 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:40:10,004 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:40:10,004 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:40:10,005 DEBUG   SenderThread:28048 [sender.py:send():380] send: history
2024-03-13 20:40:10,005 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:40:10,006 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:40:10,143 DEBUG   SenderThread:28048 [sender.py:send():380] send: exit
2024-03-13 20:40:10,144 INFO    SenderThread:28048 [sender.py:send_exit():585] handling exit code: 0
2024-03-13 20:40:10,144 INFO    SenderThread:28048 [sender.py:send_exit():587] handling runtime: 133
2024-03-13 20:40:10,145 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:40:10,145 INFO    SenderThread:28048 [sender.py:send_exit():593] send defer
2024-03-13 20:40:10,145 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:40:10,145 INFO    HandlerThread:28048 [handler.py:handle_request_defer():170] handle defer: 0
2024-03-13 20:40:10,145 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: defer
2024-03-13 20:40:10,145 INFO    SenderThread:28048 [sender.py:send_request_defer():609] handle sender defer: 0
2024-03-13 20:40:10,146 INFO    SenderThread:28048 [sender.py:transition_state():613] send defer: 1
2024-03-13 20:40:10,146 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:40:10,146 INFO    HandlerThread:28048 [handler.py:handle_request_defer():170] handle defer: 1
2024-03-13 20:40:10,146 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: defer
2024-03-13 20:40:10,146 INFO    SenderThread:28048 [sender.py:send_request_defer():609] handle sender defer: 1
2024-03-13 20:40:10,146 INFO    SenderThread:28048 [sender.py:transition_state():613] send defer: 2
2024-03-13 20:40:10,146 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:40:10,146 INFO    HandlerThread:28048 [handler.py:handle_request_defer():170] handle defer: 2
2024-03-13 20:40:10,146 INFO    HandlerThread:28048 [system_monitor.py:finish():190] Stopping system monitor
2024-03-13 20:40:10,153 DEBUG   SystemMonitor:28048 [system_monitor.py:_start():166] Finished system metrics aggregation loop
2024-03-13 20:40:10,154 INFO    HandlerThread:28048 [interfaces.py:finish():202] Joined cpu monitor
2024-03-13 20:40:10,154 DEBUG   SystemMonitor:28048 [system_monitor.py:_start():170] Publishing last batch of metrics
2024-03-13 20:40:10,154 INFO    HandlerThread:28048 [interfaces.py:finish():202] Joined disk monitor
2024-03-13 20:40:10,193 INFO    HandlerThread:28048 [interfaces.py:finish():202] Joined gpu monitor
2024-03-13 20:40:10,193 INFO    HandlerThread:28048 [interfaces.py:finish():202] Joined memory monitor
2024-03-13 20:40:10,193 INFO    HandlerThread:28048 [interfaces.py:finish():202] Joined network monitor
2024-03-13 20:40:10,194 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: defer
2024-03-13 20:40:10,194 INFO    SenderThread:28048 [sender.py:send_request_defer():609] handle sender defer: 2
2024-03-13 20:40:10,194 INFO    SenderThread:28048 [sender.py:transition_state():613] send defer: 3
2024-03-13 20:40:10,195 DEBUG   SenderThread:28048 [sender.py:send():380] send: stats
2024-03-13 20:40:10,195 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:40:10,195 INFO    HandlerThread:28048 [handler.py:handle_request_defer():170] handle defer: 3
2024-03-13 20:40:10,195 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: defer
2024-03-13 20:40:10,195 INFO    SenderThread:28048 [sender.py:send_request_defer():609] handle sender defer: 3
2024-03-13 20:40:10,195 INFO    SenderThread:28048 [sender.py:transition_state():613] send defer: 4
2024-03-13 20:40:10,196 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:40:10,196 INFO    HandlerThread:28048 [handler.py:handle_request_defer():170] handle defer: 4
2024-03-13 20:40:10,196 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: defer
2024-03-13 20:40:10,196 INFO    SenderThread:28048 [sender.py:send_request_defer():609] handle sender defer: 4
2024-03-13 20:40:10,196 INFO    SenderThread:28048 [sender.py:transition_state():613] send defer: 5
2024-03-13 20:40:10,196 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:40:10,196 INFO    HandlerThread:28048 [handler.py:handle_request_defer():170] handle defer: 5
2024-03-13 20:40:10,197 DEBUG   SenderThread:28048 [sender.py:send():380] send: summary
2024-03-13 20:40:10,197 INFO    SenderThread:28048 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:40:10,198 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: defer
2024-03-13 20:40:10,198 INFO    SenderThread:28048 [sender.py:send_request_defer():609] handle sender defer: 5
2024-03-13 20:40:10,198 INFO    SenderThread:28048 [sender.py:transition_state():613] send defer: 6
2024-03-13 20:40:10,198 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:40:10,198 INFO    HandlerThread:28048 [handler.py:handle_request_defer():170] handle defer: 6
2024-03-13 20:40:10,198 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: defer
2024-03-13 20:40:10,198 INFO    SenderThread:28048 [sender.py:send_request_defer():609] handle sender defer: 6
2024-03-13 20:40:10,199 INFO    SenderThread:28048 [sender.py:transition_state():613] send defer: 7
2024-03-13 20:40:10,199 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:40:10,199 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:40:10,199 INFO    HandlerThread:28048 [handler.py:handle_request_defer():170] handle defer: 7
2024-03-13 20:40:10,199 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: defer
2024-03-13 20:40:10,199 INFO    SenderThread:28048 [sender.py:send_request_defer():609] handle sender defer: 7
2024-03-13 20:40:10,281 INFO    Thread-12 :28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json
2024-03-13 20:40:10,689 INFO    SenderThread:28048 [sender.py:transition_state():613] send defer: 8
2024-03-13 20:40:10,689 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:40:10,689 INFO    HandlerThread:28048 [handler.py:handle_request_defer():170] handle defer: 8
2024-03-13 20:40:10,690 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: defer
2024-03-13 20:40:10,690 INFO    SenderThread:28048 [sender.py:send_request_defer():609] handle sender defer: 8
2024-03-13 20:40:10,690 INFO    SenderThread:28048 [job_builder.py:build():281] Attempting to build job artifact
2024-03-13 20:40:10,691 INFO    SenderThread:28048 [job_builder.py:_get_source_type():404] no source found
2024-03-13 20:40:10,691 INFO    SenderThread:28048 [sender.py:transition_state():613] send defer: 9
2024-03-13 20:40:10,691 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:40:10,691 INFO    HandlerThread:28048 [handler.py:handle_request_defer():170] handle defer: 9
2024-03-13 20:40:10,691 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: defer
2024-03-13 20:40:10,691 INFO    SenderThread:28048 [sender.py:send_request_defer():609] handle sender defer: 9
2024-03-13 20:40:10,691 INFO    SenderThread:28048 [dir_watcher.py:finish():358] shutting down directory watcher
2024-03-13 20:40:11,146 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:40:11,288 INFO    SenderThread:28048 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log
2024-03-13 20:40:11,289 INFO    SenderThread:28048 [dir_watcher.py:finish():388] scan: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files
2024-03-13 20:40:11,289 INFO    SenderThread:28048 [dir_watcher.py:finish():402] scan save: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\conda-environment.yaml conda-environment.yaml
2024-03-13 20:40:11,290 INFO    SenderThread:28048 [dir_watcher.py:finish():402] scan save: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\config.yaml config.yaml
2024-03-13 20:40:11,292 INFO    SenderThread:28048 [dir_watcher.py:finish():402] scan save: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\output.log output.log
2024-03-13 20:40:11,294 INFO    SenderThread:28048 [dir_watcher.py:finish():402] scan save: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\requirements.txt requirements.txt
2024-03-13 20:40:11,297 INFO    SenderThread:28048 [dir_watcher.py:finish():402] scan save: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-metadata.json wandb-metadata.json
2024-03-13 20:40:11,297 INFO    SenderThread:28048 [dir_watcher.py:finish():402] scan save: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203753-wwg0j0vs\files\wandb-summary.json wandb-summary.json
2024-03-13 20:40:11,299 INFO    SenderThread:28048 [sender.py:transition_state():613] send defer: 10
2024-03-13 20:40:11,300 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:40:11,300 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:40:11,300 INFO    HandlerThread:28048 [handler.py:handle_request_defer():170] handle defer: 10
2024-03-13 20:40:11,302 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: defer
2024-03-13 20:40:11,302 INFO    SenderThread:28048 [sender.py:send_request_defer():609] handle sender defer: 10
2024-03-13 20:40:11,302 INFO    SenderThread:28048 [file_pusher.py:finish():175] shutting down file pusher
2024-03-13 20:40:12,146 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:40:12,147 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:40:13,149 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:40:13,150 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:40:14,152 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:40:14,152 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:40:15,158 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:40:15,158 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:40:16,163 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:40:16,164 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:40:17,167 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:40:17,167 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:40:18,173 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:40:18,174 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:40:19,187 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:40:19,187 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:40:19,188 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:40:20,195 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:40:20,196 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:40:21,209 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:40:21,210 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:40:22,225 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:40:22,225 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:40:23,228 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:40:23,229 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:40:23,547 ERROR   wandb-upload_2:28048 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/wwg0j0vs/config.yaml?Expires=1710420014&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=rtKA8Nc5uKjCQQSvUwOBkeJ6yl0EzkGVehMaXlyNtn9wgWA4gjnwZhru5rghYxYdD3mBGj%2FPFA94jLF5gyE%2BjEr7JDUYu1C9VfDfOnyIymnEaAg74K0lEWH2ptNkgQDhWx74aRdu05FQjrjTj8zH9Rs6e2R%2F0FN%2FHncXM0F9%2BGTr4y9wOIdMYEfAWZeq95ZJ0LxClXJuxYvpzhkxZZVpfhi8%2FHcqA%2FpJEwQagAChQsBiJjbm9Mwh08JOQnN6ArFnFi8SFNmzcaDG3tDijvBqaKPAxF2gt7vJfN%2FO0prbKrhfZeurbl1%2FXJtJ%2FAfaHCN%2FR0X9i7aWWdvJRRPwjHtv8w%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/wwg0j0vs/config.yaml?Expires=1710420014&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=rtKA8Nc5uKjCQQSvUwOBkeJ6yl0EzkGVehMaXlyNtn9wgWA4gjnwZhru5rghYxYdD3mBGj%2FPFA94jLF5gyE%2BjEr7JDUYu1C9VfDfOnyIymnEaAg74K0lEWH2ptNkgQDhWx74aRdu05FQjrjTj8zH9Rs6e2R%2F0FN%2FHncXM0F9%2BGTr4y9wOIdMYEfAWZeq95ZJ0LxClXJuxYvpzhkxZZVpfhi8%2FHcqA%2FpJEwQagAChQsBiJjbm9Mwh08JOQnN6ArFnFi8SFNmzcaDG3tDijvBqaKPAxF2gt7vJfN%2FO0prbKrhfZeurbl1%2FXJtJ%2FAfaHCN%2FR0X9i7aWWdvJRRPwjHtv8w%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001F7BEB91600>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:40:23,547 ERROR   wandb-upload_2:28048 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '456'}
2024-03-13 20:40:23,548 ERROR   wandb-upload_2:28048 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:40:24,130 ERROR   wandb-upload_3:28048 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/wwg0j0vs/output.log?Expires=1710420014&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=pkzIjtJIFhTxfnvrfJekxyrQ2SUcFAHQZrM4N3NzmFJoE7sTDI52pTP39fdhLZ6VPi4pEN%2FXUL0Fuw7vfNAIRnOi7q%2F1%2BQIxZXn12U418GoiG3P1Gatc7cXGKotBoGqO9uV2ZEKCZk%2FUHgTcV9ox%2F9pOMC%2F9O%2BHyREB3Zg%2BYrwOrq925aKzEo3HItYoPKuWuSPlrXPWb%2FL3B%2F7UBsBVYDetQEzKVi5xTUy40LuqSD2AqLTJgO2ef6bWFVoTmkQxQ1YMw5cbxOvSsQjGNXyBhQ3Q1mQA1GqaC5uDyse3XTgd1FT53nr0RMaSW74l2KgO%2FmG9e44H3dHP5I8Ae32FKBw%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/wwg0j0vs/output.log?Expires=1710420014&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=pkzIjtJIFhTxfnvrfJekxyrQ2SUcFAHQZrM4N3NzmFJoE7sTDI52pTP39fdhLZ6VPi4pEN%2FXUL0Fuw7vfNAIRnOi7q%2F1%2BQIxZXn12U418GoiG3P1Gatc7cXGKotBoGqO9uV2ZEKCZk%2FUHgTcV9ox%2F9pOMC%2F9O%2BHyREB3Zg%2BYrwOrq925aKzEo3HItYoPKuWuSPlrXPWb%2FL3B%2F7UBsBVYDetQEzKVi5xTUy40LuqSD2AqLTJgO2ef6bWFVoTmkQxQ1YMw5cbxOvSsQjGNXyBhQ3Q1mQA1GqaC5uDyse3XTgd1FT53nr0RMaSW74l2KgO%2FmG9e44H3dHP5I8Ae32FKBw%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001F7BEB93310>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:40:24,130 ERROR   wandb-upload_3:28048 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1448'}
2024-03-13 20:40:24,131 ERROR   wandb-upload_3:28048 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:40:24,234 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:40:24,235 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:40:24,235 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:40:24,461 ERROR   wandb-upload_1:28048 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/wwg0j0vs/conda-environment.yaml?Expires=1710420014&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Q8AJVxsczGQw3vMheXbswEKxahfkSS2dS2TPqDdCLgd558TEpXsHQGz9%2FJRxk%2B5BBAM2D6Ij1xUWn1EuVkjwcLwC0TxLD%2B%2B629U0Um7dYcxy9P7xg5XDu5AeC%2BOabQlpZN%2F8RYl0llfayVtI4s6vCpk7eF0shFUT4Flh%2Bu0scXY0FIMlbG8FhN6sW9c%2BIMo44He4QmfgcGNKKtEa1fE08NGuo2eMzQwrnuY7eRcp3qLTj6kmtWFV%2FalsB4H0oK%2FkTrmiXRq8QkEi50E7BzUP5V8PYASYNV9FbJ%2Fx%2FDtiEyiVZJCKvJJxnmVqXoHBNHj%2FtJfRmFqwmE13QK%2FeWQIsPA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/wwg0j0vs/conda-environment.yaml?Expires=1710420014&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Q8AJVxsczGQw3vMheXbswEKxahfkSS2dS2TPqDdCLgd558TEpXsHQGz9%2FJRxk%2B5BBAM2D6Ij1xUWn1EuVkjwcLwC0TxLD%2B%2B629U0Um7dYcxy9P7xg5XDu5AeC%2BOabQlpZN%2F8RYl0llfayVtI4s6vCpk7eF0shFUT4Flh%2Bu0scXY0FIMlbG8FhN6sW9c%2BIMo44He4QmfgcGNKKtEa1fE08NGuo2eMzQwrnuY7eRcp3qLTj6kmtWFV%2FalsB4H0oK%2FkTrmiXRq8QkEi50E7BzUP5V8PYASYNV9FbJ%2Fx%2FDtiEyiVZJCKvJJxnmVqXoHBNHj%2FtJfRmFqwmE13QK%2FeWQIsPA%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001F7BEAEA7A0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:40:24,461 ERROR   wandb-upload_1:28048 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '9492'}
2024-03-13 20:40:24,462 ERROR   wandb-upload_1:28048 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:40:24,727 ERROR   wandb-upload_4:28048 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/wwg0j0vs/requirements.txt?Expires=1710420014&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=TvCarPEYmfzV%2F0utcYGbp9KWutR05GyxZ%2FLkGX4Gp2e%2F28roHtEy6fNITag9V5yJ%2FjlCs5GxjDideDhbTblEzySmO%2By4axUnD503J5264XhAYnGOXSzoaYaIUvFfN4FbwZbe76VCqETEY78DxCE9MEIeQ0jUq%2FHQMJIuyctKhnctfTIjfO8J6F%2FS1bAitfxrxJSgQGULi4gWkmFFmqRb%2Ffte7BjcuPSenKpqc7P7maSJUqsYCROI4h9klhJsu0SII38QhSbKF%2B78sW8IlSH0LVkALHwAXw9YPLXAw1Y3fq0J%2FfmnlYmnHktKQmKfPKhwBeAaTSKNvPoJbMEGBpp%2FTg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/wwg0j0vs/requirements.txt?Expires=1710420014&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=TvCarPEYmfzV%2F0utcYGbp9KWutR05GyxZ%2FLkGX4Gp2e%2F28roHtEy6fNITag9V5yJ%2FjlCs5GxjDideDhbTblEzySmO%2By4axUnD503J5264XhAYnGOXSzoaYaIUvFfN4FbwZbe76VCqETEY78DxCE9MEIeQ0jUq%2FHQMJIuyctKhnctfTIjfO8J6F%2FS1bAitfxrxJSgQGULi4gWkmFFmqRb%2Ffte7BjcuPSenKpqc7P7maSJUqsYCROI4h9klhJsu0SII38QhSbKF%2B78sW8IlSH0LVkALHwAXw9YPLXAw1Y3fq0J%2FfmnlYmnHktKQmKfPKhwBeAaTSKNvPoJbMEGBpp%2FTg%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001F7BEB93400>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:40:24,728 ERROR   wandb-upload_4:28048 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '3463'}
2024-03-13 20:40:24,728 ERROR   wandb-upload_4:28048 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:40:25,238 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:40:25,239 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:40:25,898 ERROR   wandb-upload_5:28048 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/wwg0j0vs/wandb-summary.json?Expires=1710420015&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=YLF%2FrMnrGPjmpX4QEF2QEqDmPl0%2Bj%2FYpO9Dt7GUFmNC%2BFPu%2F5q%2B8ssgH4YVm%2FQ5kXUpax6gmKJ3ypdDOqX%2Fp9i9e3pW%2Bx6Gb%2F1Ih9eMrT5yvRfq7yHpzJc5JZLye95n4qsJjee8R1j4O6L7keXEm0EB0Q1CvFe6OwqJJZL9Uu6w%2FnzUDwEsoNhW9suRxpa8vZG4F6n2EFdFAMZaQxOcCvPxpIXwmehMHi40m%2F%2Bji4snSiDma1PkdMEEyoQn7xwK6DjpKcPYFBg08eofq5MLJDUVwISWMAttQy10QQWrDUjW8%2Bu8db6Lwty4wr7Xya5d0SwUxTUj6uJ%2B8S4e6MJgTmw%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/wwg0j0vs/wandb-summary.json?Expires=1710420015&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=YLF%2FrMnrGPjmpX4QEF2QEqDmPl0%2Bj%2FYpO9Dt7GUFmNC%2BFPu%2F5q%2B8ssgH4YVm%2FQ5kXUpax6gmKJ3ypdDOqX%2Fp9i9e3pW%2Bx6Gb%2F1Ih9eMrT5yvRfq7yHpzJc5JZLye95n4qsJjee8R1j4O6L7keXEm0EB0Q1CvFe6OwqJJZL9Uu6w%2FnzUDwEsoNhW9suRxpa8vZG4F6n2EFdFAMZaQxOcCvPxpIXwmehMHi40m%2F%2Bji4snSiDma1PkdMEEyoQn7xwK6DjpKcPYFBg08eofq5MLJDUVwISWMAttQy10QQWrDUjW8%2Bu8db6Lwty4wr7Xya5d0SwUxTUj6uJ%2B8S4e6MJgTmw%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001F7BEBB7670>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:40:25,898 ERROR   wandb-upload_5:28048 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '207'}
2024-03-13 20:40:25,898 ERROR   wandb-upload_5:28048 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:40:26,248 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:40:26,248 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:40:27,254 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:40:27,254 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:40:28,257 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:40:28,258 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:40:29,264 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:40:29,264 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:40:29,265 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:40:30,267 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:40:30,268 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:40:31,279 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:40:31,280 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:40:32,294 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:40:32,295 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:40:33,306 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:40:33,306 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:40:34,316 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:40:34,317 DEBUG   HandlerThread:28048 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:40:34,317 DEBUG   SenderThread:28048 [sender.py:send_request():407] send_request: poll_exit
